R/select_netZoo.R

```diff
@@ -1,56 +1,104 @@
 #' @title Select NetZooR Algorithm
 #' @description A simple function to run NetZooR algorithm of choice
-#' @param algorithm (Character) Choose between panda, lioness, condor, alpaca, and monster
-#' @param geneExp (Object) Name of expression data frame object generated using read_gene() or filter_gene()
-#' @param basePath (Character) Gene expression base group file path
-#' @param compPath (Character) Gene expression comparison group file path
-#' @param startSample (Numeric) Value to indicate lionessPy first sample - default is 1
-#' @param endSample (Numeric) Value to indicate lionessPy last sample - default is 'None'
+#' @param algorithm (Character) Choose between 'pandaPy', 'lionessPy', 'condor', 'alpaca', and 'monster'
+#' @param expr_file (Character) Gene expression base group filepath
+#' @param expr (Object) Name of expression data.frame object generated using `read_gene()` or `filter_gene()`
+#' @param motif_file (Character) value indicating which motif data to use.  Options include NULL for Pearson correlation
+#' matrix, 'default' for built-in data, or a user-specified filepath.
+#' @param ppi_file (Character) value indicating which protein-protein interaction data to use.  Options include
+#' NULL for no additional interaction, 'default' for built-in data, or a user-specified filepath.
+#' @param comp_file (Character) Gene expression comparison group filepath for 'alpaca' networks
+#' @param start_sample (Numeric) Value to indicate 'lionessPy' first sample - default is 1
+#' @param end_sample (Numeric) Value to indicate 'lionessPy' last sample - default is 'None'
+#' @param design (Numeric) Binary vector for 'monster' algorithm case control partition.
 #' @details In order to run alpaca or monster, split_gene() must be run first
 #' @return Returns output of selected algorithm to be used for visualization or second algorithm
 #' @author Walid Nashashibi (\url{https://github.com/wnashash/})
 #' @examples
 #' \dontrun{
 #' library(processNetZoo)
-#' exp_path <- system.file("extdata", "expression_test.csv", package = "processNetZoo", mustWork = TRUE)
+#'
+#' # -- pandaPy --
+#' exp_path <- system.file("extdata",
```

Comment:
Consider adding an example for every algorithm that you consider fully implemented since each one takes different arguments